diff --git a/train_wan_with_motion.sh b/train_wan_with_motion.sh
index 153237d..6b3c584 100755
--- a/train_wan_with_motion.sh
+++ b/train_wan_with_motion.sh
@@ -306,6 +306,7 @@ accelerate launch train_wan_with_motion.py \
     ${WARP_ARGS} \
     --wandb_project "Sim4Videos" \
     --wandb_run_name ${wandb_run_name} \
+    --use_wandb \
 
 
 if [ $? -ne 0 ]; then
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 2c1d3c6..daebf7c 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20251213_092323-7m9p38p4/logs/debug-internal.log
\ No newline at end of file
+run-20251213_093437-qwmszamx/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index c8c2902..22f6f71 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20251213_092323-7m9p38p4/logs/debug.log
\ No newline at end of file
+run-20251213_093437-qwmszamx/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 85a75dc..65c9dc7 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20251213_092323-7m9p38p4
\ No newline at end of file
+run-20251213_093437-qwmszamx
\ No newline at end of file
