diff --git a/train_wan_with_motion.sh b/train_wan_with_motion.sh
index 76b355e..bc96167 100755
--- a/train_wan_with_motion.sh
+++ b/train_wan_with_motion.sh
@@ -280,6 +280,7 @@ accelerate launch train_wan_with_motion.py \
     --depth_loss_type ${DEPTH_LOSS_TYPE} \
     --depth_scale ${DEPTH_SCALE} \
     ${SPATIOTEMPORAL_ARGS} \
+    --use_wandb \
     --wandb_project "Sim4Videos" \
     --wandb_run_name ${wandb_run_name} \
 
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 6bb3b0e..fee0572 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20251212_075021-qfo1nfwe/logs/debug-internal.log
\ No newline at end of file
+run-20251212_080303-1tjaskiy/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index f37cf00..c813d3c 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20251212_075021-qfo1nfwe/logs/debug.log
\ No newline at end of file
+run-20251212_080303-1tjaskiy/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 9faa5e3..c039236 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20251212_075021-qfo1nfwe
\ No newline at end of file
+run-20251212_080303-1tjaskiy
\ No newline at end of file
